_episodes/10-distribution.md

```diff
@@ -0,0 +1,245 @@
+---
+title: "Distributing Packages"
+teaching: 30
+exercises: 60
+questions:
+- "How can you get your project out there?"
+objectives:
+- "Discuss community strategies"
+keypoints:
+- ""
+---
+
+# Preliminary
+
+Register a test account on test.pypi.org.
+
+Create an access token, and *don't lose it!*
+(I keep a copy of the text for my token in my password manager.)
+
+Create `$HOME/.pypirc`. It should look something like the following.
+
+~~~
+[testpypi]
+  username = __token__
+  password = pypi-Abunchofbase64encodedbinarydataAbunchofbase64encodedbinarydataAbunchofbase64encodedbinarydataAbunchofbase64encodedbinarydata
+~~~
+{: .output}
+
+You will also eventually need to get an account on the main pypi.org,
+but you can do that later.
+
+Install some additional packages to help build and upload your distribution.
+~~~
+pip install build twine
+~~~
+{: .language-bash}
+
+# Preparing a release
+
+## Release notes
+
+Make sure your README file and documentation are up to date.
+Check the project description in your project's metadata, too.
+
+Consider whether there is some announcement text that you would
+want to include in a text file or in a blurb in your GitHub Releases tab.
+
+*TODO: Do we care about GitHub's Release and Tags tabs?*
```

Comment:
```suggestion
want to include in a text file or in a blurb in your GitHub
[Releases tab](https://docs.github.com/en/repositories/releasing-projects-on-github/about-releases).
This can be a good place to publish [release notes](https://docs.github.com/en/repositories/releasing-projects-on-github/automatically-generated-release-notes).
```